tools/lxdclient/utils.go

```diff
@@ -65,6 +66,8 @@ func IsRunningLocally() (bool, error) {
 	return running, nil
 }
 
+const errIPV6NotSupported = `cannot listen on https socket: listen tcp \[::\]:\d*: socket: address family not supported by protocol`
```

Comment:
The "socket: address family not supported by protocol" part is effectively considered Linux API (glibc) and so is very unlikely to change so long as LXD doesn't mask the error.

The rest of the string comes from a mix of underlying golang packages and LXD, and I'd strongly recommend not relying on that part. So checking if "socket: address family not supported by protocol" is part of the error string should work fine and be much more reliable.